laravel/app/Http/Controllers/SlotController.php

```diff
@@ -100,6 +102,14 @@ public function take(SlotRequest $request, Slot $slot)
         // Has somebody else already taken this slot?
         if(is_null($slot->user))
         {
+            $concurrent_slot_warning = $this->warnIfConcurrentSlotForUserExists($request, $slot, Auth::user());
+            if($concurrent_slot_warning)
+            {
+                return back();
+            }
+
+
+
```

Comment:
you got a lot of extra space, usually we don't leave extra lines out like that, try to keep things neat.